packages/tslint-config/src/rules/blueprintClassesConstantsRule.ts

```diff
@@ -5,7 +5,7 @@
  */
 
 import * as Lint from "tslint";
-import * as utils from "tsutils";
+import { isExpressionStatement, isJsxAttribute, isPropertyAssignment, isStringLiteral } from "tsutils/typeguard/2.8";
```

Comment:
Did `tsutils` change how it's exporting things? Possible to avoid the submodule import?

Reply:
no, since tsutils references types from different tsc versions, and this is the best way to deal with it